desktop/apps/article2/components/InfiniteScrollArticle.js

```diff
@@ -43,35 +44,55 @@ export default class InfiniteScrollArticle extends React.Component {
         })
       })
 
-      this.setState({
-        articles: articles.concat(data.articles),
-        isLoading: false,
-        offset: offset + 3
-      })
+      // TODO:
+      // At some point this could go in a query so as not to fetch unnecessary data
+
+      // Ignore 'Featured'
+      const newArticles = data.articles.filter(article => !article.featured)
```

Comment:
Ah I can see where this might be confusing -- we want to exclude the `feature` layout here and not the `featured` flag. Soo we'll want something like `article.layout !== 'feature'`

Reply:
ah! will do.

Reply:
Updated, sorry for the delay